op-dispute-mon/mon/service.go

```diff
@@ -157,14 +165,16 @@ func (s *Service) initBonds() {
 }
 
 func (s *Service) initOutputRollupClient(ctx context.Context, cfg *config.Config) error {
-	if cfg.RollupRpc == "" {
+	if len(cfg.RollupRpc) == 0 {
 		return nil
 	}
-	outputRollupClient, err := dial.DialRollupClientWithTimeout(ctx, dial.DefaultDialTimeout, s.logger, cfg.RollupRpc)
-	if err != nil {
-		return fmt.Errorf("failed to dial rollup client: %w", err)
+	for _, rpc := range cfg.RollupRpc {
+		client, err := dial.DialRollupClientWithTimeout(ctx, dial.DefaultDialTimeout, s.logger, rpc)
```

Comment:
hmm, just realised this will mean op-dispute-mon fails to startup if any of the rollup nodes are offline. We should be able to avoid that by using lazy dial:

```suggestion
		client, err := dial.DialRollupClientWithTimeout(ctx, dial.DefaultDialTimeout, s.logger, rpc, client.WithLazyDial())
```

Probably worth testing manually though to make sure that we can startup even if one endpoint is unavailable.

Reply:
Done, not manually tested yet.